### PR TITLE
Fix bugs on ligand handling and pdbqt ligand support

### PR DESCRIPTION
**Pull request by JoseAntonioSiguenza**
## Description

Current `pose_generation` deepchem support with Vina was causing issues for some reasons:
1. Two-letter atoms (such as Cl, Br, etc) were transformed to mayus (CL, BR) which caused Vina to break
2. PDBQT ligand files weren't able to be processed.

## Type of change

Please check the option that is related to your PR.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [x] Run `mypy -p deepchem` and check no errors
  - [x] Run `flake8 <modified file> --count` and check no errors
  - [x] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by rbharath — COMMENTED** [review]
@JoseAntonioSiguenza Let's not add meeko as a dependency but instead pull in the code we need into utils and cite them

@shreyasvinaya Please review this PR as well

**Review by shreyasvinaya — APPROVED** [review]
LGTM

cc: @rbharath

**Comment by JoseAntonioSiguenza** [comment]
@shreyasvinaya @rbharath I need to do one more pass on pdbqt utils and fix mypy. I'll let you know when is ready for review

**Comment by JoseAntonioSiguenza** [comment]
@rbharath @shreyasvinaya PR ready for review

File: deepchem/dock/tests/test_pose_generation.py [hunk 1]
```diff
@@ -176,8 +176,38 @@ def test_pocket_vina_poses(self):
...
+
+        # Note this may download autodock Vina...
+        vpg = dc.dock.VinaPoseGenerator()
+        centroid = np.array([15.190, 53.903, 16.917])
+        box_dims = np.array([20.0, 20.0, 20.0])
+        with tempfile.TemporaryDirectory() as tmp:
+            scores = vpg.generate_poses((protein_file, ligand_file),
+                                        centroid=centroid,
+                                        box_dims=box_dims,
+                                        exhaustiveness=5,
```
Comment (rbharath) [inline deepchem/dock/tests/test_pose_generation.py:207]: @JoseAntonioSiguenza Let's lower exhaustiveness to speed up this test

File: deepchem/utils/test/test_pdbqt_utils.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def setUp(self):
         self.protein_file = os.path.join(current_dir,
                                          "../../dock/tests/1jld_protein.pdb")
         self.ligand_file = os.path.join(current_dir,
-                                        "../../dock/tests/1jld_ligand.sdf")
+                                        "../../dock/tests/Ochratoxin A.sdf")
```
Comment (rbharath) [inline deepchem/utils/test/test_pdbqt_utils.py:15]: Please remove and use 1jld ligand

**Review by rbharath — COMMENTED** [review]
Some small test fixes and we can merge in after

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**